CefGlue.Common/CefRuntimeLoader.cs

```diff
@@ -58,6 +58,19 @@ private static void InternalInitialize(CefSettings settings = null, KeyValuePair
                     settings.FrameworkDirPath = basePath;
                     settings.ResourcesDirPath = resourcesPath;
                     break;
+                
+                case CefRuntimePlatform.Linux:
+                    string localesPath = Path.Combine(basePath, "locales");
+                    if (Directory.Exists(localesPath))
+                        settings.LocalesDirPath = localesPath;
```

Comment:
to make it consistent with the remaining codebase
```suggestion
                    var localesPath = Path.Combine(basePath, "locales");
                    if (Directory.Exists(localesPath))
                    {
                        settings.LocalesDirPath = localesPath;
                    }
```